projects/plugins/paypal-payment-buttons/readme.txt

```diff
@@ -1,70 +1,90 @@
 === PayPal Payment Buttons ===
-Contributors: automattic,
-Tags: paypal, payments, ecommerce, buttons
+Contributors: paypal,automattic,woocommerce
+Tags: paypal, payments, ecommerce, blocks, checkout
 Requires at least: 6.7
 Requires PHP: 7.2
 Tested up to: 6.8
-Stable tag: 0.1.0-alpha
+Stable tag: 0.1.0
```

Comment:
This will be automatically edited when the plugin is first released. You cannot edit it in this PR.